config/clusters/2i2c/staging.values.yaml

```diff
@@ -23,6 +30,13 @@ jupyterhub:
           name: 2i2c
           url: https://2i2c.org
   hub:
+    services:
+      dex:
```

Comment:
We don't need this service to be visible from the `services` menu right?
What do you think about using `display: false` to hide it (the same we're doing for dask-gateway)?

I think it might be confusing for users to have services throwing 404s

Reply:
Good call, I've updated it @GeorgianaElena!